app/controllers/UploadController.scala

```diff
@@ -78,7 +78,10 @@ class UploadController(override val authActions: HMACAuthActions, awsConfig: AWS
       projectId = atom.plutoData.flatMap(_.projectId),
       s3Key = CompleteUploadKey(awsConfig.userUploadFolder, id).toString,
       assetVersion = -1,
-      atomId = atom.id
+      atomId = atom.id,
+      title = atom.title,
+      user = user.email,
+      posterImageUrl = atom.posterImage.flatMap(_.master).map(_.file)
```

Comment:
Why these new properties? As far as I can tell, we're not using them?

Reply:
i am using them at the pluto end, to make it easier to find masters if one does not make it through or the "launch detector" phase does not work.  They are what we are likely to want to search on.